backend/Makefile

```
@@ -254,6 +256,53 @@ test-fuzz: ## Run fuzz tests (REST API and GraphQL)
+	@docker run \
+		--platform $(CFL_PLATFORM) \
+		--rm \
+		-e ARCH=x86_64 \
+		-e FUZZING_ENGINE=libfuzzer \
+		-e OUT=/out \
+		-e SANITIZER=$(CFL_SANITIZER) \
+		-v $(CFL_OUT_DIR):/out \
+		$(CFL_IMAGE) \
+		bash build.sh
+
+test-cluster-fuzz-lite: build-cluster-fuzz-lite-fuzzers ## Run ClusterFuzzLite fuzz targets locally
```

checkmake (0.3.2)

[warning] Target body for "test-cluster-fuzz-lite" exceeds allowed length of 5 lines (13).

(maxbodylength)